{{ cookiecutter.dir_name }}/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
+    }
+}
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
```

Comment:
I don't think gradle will download the android tools - I had 25.0.1 when I started playing with this template, and gradle failed with a hard error - it took me a good while to work out what exactly what needed to be installed.

I can think of two possible approaches here:

* Make the tools version templated, and have Briefcase discover the tools version.

* Hard code the tools version in the template, and add a step into Briefcase to update the Android tools so that this version is available. 

However, this doesn't need to be solved right now; in the short term, we can live with a hard coded version, and document exactly what is needed to run the template.